html/mycakeapp/src/Controller/MembersController.php

```diff
@@ -103,4 +109,26 @@ public function delete($id = null)
 
         return $this->redirect(['action' => 'index']);
     }
+    public function create()
+    {
+        $this->viewBuilder()->setLayout('frame-title');
+        $entity = $this->Members->newEntity();
+        if ($this->request->is('post')) {
+            $entity = $this->Members->patchEntity($entity, $this->request->getData());
+            $entity["total_point"] = 0;
+            $entity["is_deleted"] = 0;
+            $entity["is_provisional"] = 1;
```

Comment:
```suggestion
            $entity["is_provisional"] = 0;
```
本来は仮登録フラグのためデフォルト1ですが、今回はメール等を使った本登録チェックほ行わないのでデフォルト0(本登録済み)にして良いと思います。

Reply:
確かにそうですね...!!修正します！